app/controllers/dashboard_controller.rb

```diff
@@ -224,7 +224,20 @@ def landing_page
 
   def browse_tag
     @tag = Tag.find_by(ai_text: params[:ai_text])
-    @collections = @tag.collections.unrestricted.has_intro_block.has_picture.not_empty
+    collections = @tag.collections
+                      .includes(:owner, { works: :work_statistic })
+                      .unrestricted
```

Comment:
Imagine that you have a public document set from a private collection, as happens with the `cwrgm` user.  You would like the document set to appear in the list.

The `unrestricted` scope will prevent the public document set from appearing in the tag cloud.